staker/stateless_block_validator.go

```diff
@@ -112,6 +112,39 @@ func GlobalStatePositionsAtCount(
 	return startPos, GlobalStatePosition{batch, posInBatch + 1}, nil
 }
 
+func FindBatchContainingMessageIndex(
```

Comment:
I think this function just exists somewhere else now. We should look around for it to make sure we aren't duplicating code.